unit-tests/Extension/BuiltIn/StringMethodTest.php

```diff
@@ -23,6 +23,105 @@
 
 class StringMethodTest extends \PHPUnit_Framework_TestCase
 {
+    public function testCamelize()
```

Comment:
You should use data provider (https://phpunit.de/manual/current/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers)